concepts/classes/about.md

```diff
@@ -0,0 +1,235 @@
+# classes
+
+Classes are a central concept in [object-oriented programming][object-oriented-programming].
+[In Crystal, everything is an object][everything-is-an-object], every object has a type, and it can respond to some methods.
+Classes are blueprints to create objects, providing initial values for state, like variables, and implementations of behavior, like methods.
+Objects are an instance of a class, that hold all the instance variables and states.
+
+## Class definition
+
+A class is defined using the `class` keyword, followed by the name of the class written in `UpperCamelCase`.
+
+```crystal
+class Account
+end
+```
+
+## Creating an instance
+
+To [create an instance of a class][new-initialize], you call the class name with the `new` method.
+
+```crystal
+class Account
+end
+
+account = Account.new
+```
+
+In the example above the variable `account` is an instance of the class `Account`.
+
+When creating an instance, there is a special method called `initialize` that acts as a constructor.
+It is called when an instance is created.
+The constructor allows you to set the initial state of the instance.
+The `initialize` method can take arguments, which are passed when creating an instance.
+
+```crystal
+class Account
+  def initialize(number)
+    5 + number # => 9
+  end
+end
+
+Account.new(4)
+```
+
+````exercism/note
+The `initialize` method cannot manually return a value, instead, it returns the newly created instance.
```

Comment:
This is technically not accurate, but I'm not sure how to reword to make it true but not confusing for new learners. Maybe:

```suggestion
The `initialize` method cannot manually return a value, instead, `new` returns the newly created instance.
```